modules/trusted-firmware-m/Kconfig

```diff
@@ -20,6 +20,17 @@ config TFM_BL2
 	bool
 	default n
 
+config TFM_SECURE_UART1
+	bool "Use UART1 for TF-M logging"
+	default y
+	depends on !UART_1_NRF_UARTE
+	depends on SOC_NRF5340_CPUAPP || SOC_NRF9160
+	help
+	  Reserve the UART1 port for TF-M logging. This makes the UART1
+	  peripheral unavailable to the non-secure application. When this
+	  option is selected the device tree node for UART1 needs to be
+	  disabled for the non non-secure application.
```

Comment:
non non-secure application.

Reply:
i.e. secure? 😆

Reply:
I should just add another non and it should be correct :D Fixed ;)